test_unstructured_ingest/.gitignore

```diff
@@ -1,2 +1,4 @@
 structured-output
 download
+test_unstructured_ingest/expected-text-output/**
```

Comment:
From the folder name it sounds like this is where the text output fixtures would go. Don't we want to track changes in git for these fixtures?

Reply:
there are not text fixtures so far, only json fixtures. this PR derives the clean text from json's at runtime.